netci.groovy

```diff
@@ -75,7 +75,7 @@ def targetGroupOsMapInnerloop = ['netcoreapp': ['Windows_NT', 'Ubuntu14.04', 'Ub
         // Archive results.
         Utilities.addArchival(newJob, '**/coverage/*,msbuild.log')
         // Timeout. Code coverage runs take longer, so we set the timeout to be longer.
-        Utilities.setJobTimeout(newJob, 180)
+        Utilities.setJobTimeout(newJob, 720)
```

Comment:
Does the number reflect minutes? I am just wondering why we are increasing the timeout 4x? should we have a more reasonable timeout? or we don't really care?

Reply:
:)

> "Increasing the timeout 4 fold to get a few runs and access the appropriate value." 

Let's see how it goes, my guess is that 6 hours should be enough but I want a few runs before making a choice.